pkg/operator/ceph/config/store.go

```diff
@@ -100,6 +126,39 @@ func (s *Store) CreateOrUpdate(clusterInfo *cephconfig.ClusterInfo) error {
 	return nil
 }
 
+func (s *Store) applyLegacyOverrides(toFile *ini.File) error {
+	ovrTxt := []byte(s.overrideConfig())
+	fmt.Printf("\ntoFile: %+v\n\n", toFile)
```

Comment:
can we write this with `logger` instead of `fmt`?

Reply:
Haha! Oops. That's just from my testing. Pay no attention to the man behind the curtain.